pipeline/packer/jenkins-master/jenkins-master.json

```diff
@@ -0,0 +1,51 @@
+{
+  "_description": "This template produces a Jenkins server.",
+  "variables": {
+    "repository": "{{env `DOCKER_REPOSITORY`}}",
+    "tag": "{{env `BUILD_TAG`}}",
+    "docker_registry_url": "{{env `DOCKER_REGISTRY_URL`}}"
+  },
+  "builders": [
+    {
+      "type": "docker",
+      "image": "jenkins/jenkins:lts",
+      "commit": true,
```

Comment:
Is it worth using `pull: false` here as well?

Reply:
I don't think so, as we actually do want to pull that image from Docker hub. It was only needed in the jenkins worker because I ended up splitting the packer script into a dockerfile and a packer script, I haven't had to do that for the Jenkins master.